### PR TITLE
WL-4711: Calendar not perfectly responsive

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/css/Simplepagetool.css [hunk 1]
```diff
@@ -1449,9 +1449,6 @@ padding:1px;
 .edit-col span {
 color: #666;
 }
-.calendar-div{
-width: 550px;
```
Comment (buckett) [inline lessonbuilder/tool/src/webapp/css/Simplepagetool.css:1453]: Do we want to make this a max-width?

Comment (buckett) [reply lessonbuilder/tool/src/webapp/css/Simplepagetool.css:1453]: @nicholaswilson100 Do we want the calendar to fill the whole space?

**Comment by nicholaswilson100** [comment]
Yes I need to change this PR

## 

Nick Wilson
Software Developer / Project Lead
Technology Enhanced Learning, Academic IT
IT Services, University of Oxford
13 Banbury Road, OX2 6NN
Tel: 01865 613716

**Comment by nicholaswilson100** [comment]
I've made tihs max width

**Pull request merged**